tests/New-DbaDatabase.Tests.ps1

```diff
@@ -14,6 +14,15 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
 }
 
 Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
+    BeforeAll {
+        $dbname = 'dbatoolsci_newdb', 'dbatoolsci_newdb1', 'dbatoolsci_newdb2'
+        $null = Get-DbaDatabase -SqlInstance $script:instance2, $script:instance3 -Database $dbname | Remove-DbaDatabase -Confirm:$false
+    }
+
+    AfterAll {
+        $null = Get-DbaDatabase -SqlInstance $script:instance2, $script:instance3 -Database $dbname | Remove-DbaDatabase -Confirm:$false
+    }
```

Comment:
The test is failing so I'm not sure if it is because of this additional, but please review and correct accordingly.

We already clean up all 4 databases within each `It` block so I'm not sure why the `BeforeAll` and `AfterAll` is needed, it only covers 3 of the 4 databases created.

Reply:
My bad, had some bug in the constants.local file.